### DISCUSSION TITLE
unable to access Web UI of security onion

### DISCUSSION THREAD

**Discussion by opindersingh** [Security-Onion-Solutions/securityonion#5496] [Unsupported Versions]
I have configured NAT and bridge adapters in VM. So-status is also OK for all but I am unable to access security onion web UI. Ping is working fine from security onion VM to my windows host machine. can anybody help to figure out the issue?

**Comment by dougburks** [comment]
Have you tried running so-allow to allow your IP address through the firewall?
https://docs.securityonion.net/en/2.3/so-allow.html#so-allow
